.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
```

Comment:
@paolobueno its nice but dont think its important. Better not to put it one by one but create some task and then update all modules and apps using it